test/system/002-help.bats

```diff
@@ -63,61 +63,61 @@ setup() {
 
   assert_failure
   assert_line --index 0 "Error: unknown flag: --foo"
-  assert_line --index 1 "Run 'toolbox --help' for usage."
+  assert_line --index 1 "Run 'toolbox create --help' for usage."
```

Comment:
I actually want to keep it as:
`Run 'toolbox --help' for usage`

There can be several reasons why the command line parsing failed.  Some of these failures can be quite subtle in terms of the code paths they take and not obvious at first sight.  Hence, it's a lot easier to reason about the code if they simply suggest `toolbox --help`.  It's not worth dealing with the odd bug report arising out of a slightly wrong suggestion.

Secondly, I'd like to consolidate the mental model followed by documentation.  It's a lot easier to manage the content if we are sure that people generally start at the top-level and then dive into each command.  That's why the top level `toolbox(1)` manual is deliberately brief and prominently points out each command.

Reply:
Personally, I find it more useful for the usage to mention the specific command. But I don't have a strong opinion about this, so I'll revert that :+1:.